tests/test_filter_datasets.py

```diff
@@ -294,3 +341,109 @@ def test_stratified_crossval_splits_multiple_attributes(many_stimuli, crossval_f
     assert sum(len(f.x) for f in train_fixations) == (crossval_folds - val_folds - test_folds) * len(fixations.x)
 
     assert len(train_stimuli) == crossval_folds
+
+
+@pytest.mark.parametrize('attribute_name , attribute_value, invert_match', [('dva', 1, False)])
```

Comment:
if you pass only one set of parameters, there is no need to use `parametrize`. Simply use the values directly in the test.

Reply:
that is a personal choice. It just makes it more easily readable for me xD. But I can easily change that

Reply:
If it's on purpose, I'm fine with it :).

Reply:
i changed it for now.